src/command-line/uninstall.js

```diff
@@ -24,14 +24,20 @@ program
 		const packagesPath = Helper.getPackagesPath();
 		const packagesParent = path.dirname(packagesPath);
 		const packagesConfig = path.join(packagesParent, "package.json");
-		const packageWasNotInstalled = `${colors.green(packageName)} was not installed.`;
+		const packageWasNotInstalled = `${colors.green(packageName)} is not installed.`;
```

Comment:
Meh :man_shrugging:

Reply:
I'm also not sure on this one. Saying "x is not installed" doesn't give the indication that we want, because when you finish installing, of course it's not installed. The key point is that we *didn't uninstall* because there wasn't a package installed to uninstall. At this point, when we print this message, this statement will always be the case, so we aren't telling the user what we want.

Reply:
Also, at the *very least* please make the name of the variable the same as the string ;-)

Reply:
> At this point, when we print this message, this statement will always be the case, so we aren't telling the user what we want.

It's printed when the package is in fact not installed. What do you suggest the message should be?

Reply:
> It's printed when the package is in fact not installed.

Sure, but my point is that that's what we *want*. At the end of this command, it will *always* not be installed. That's the point of uninstalling. However, if it *wasn't* installed before, then there might be an issue with that, there might have been a typo, etc.

I would say that what we had before was probably fine. It's correct, and it makes sense.